packages/bridge-ui/src/components/form/To.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import ButtonWithTooltip from '../ButtonWithTooltip.svelte';
+  import TooltipModal from '../modals/TooltipModal.svelte';
+
+  export let showTo: boolean = false;
```

Comment:
do these need to be exported? i don't see them as imported in BridgeForm

Reply:
theyre not imported in BridgeForm, theyre bound to using `bind` value to make the values in thsi component available in the parent BridgeForm component.

Reply:
theyre not imported in BridgeForm, theyre bound to using `bind` value to make the values in thsi component available in the parent BridgeForm component.